XRTK.SDK/Packages/com.xrtk.sdk/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -1000,21 +994,21 @@ private void UpdateRigHandles()
 
                 switch (edgeAxes[i])
                 {
-                    case CardinalAxisType.X:
+                    case CardinalAxisType.Z:
```

Comment:
In this instance CardinalAxisType.Z is mapping to LinkDimensions.X ? Validate

Reply:
It's just the order was adjusted. Validated, should be correct.